framework/conf/dependencies.yml

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Why is this file needed? I don't see that it's used somewhere.

Reply:
@tomparle There is two files:
1. `framework/dependencies.yml` - it's actually used
2. `framework/conf/dependencies.yml` - it's not used. You added this file in your PR. 

Yes, file `framework/conf/dependencies.yml` is not used and should be deleted.